### PR TITLE
update analyze concurrency for statistics

### PR DESCRIPTION
**Pull request by hawkingrei**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: statistics.md [hunk 1]
```diff
@@ -383,7 +383,15 @@ ANALYZE INCREMENTAL TABLE TableName PARTITION PartitionNameList INDEX [IndexName
...
-目前 ANALYZE 执行的时候会被切分成一个个小的任务，每个任务只负责某一个列或者索引。`tidb_build_stats_concurrency` 可以控制同时执行的任务的数量，其默认值是 4。
+目前 ANALYZE 执行的时候会被切分成一个个小的任务，每个任务只负责某一个列或者索引。`tidb_build_stats_concurrency` 可以控制同时执行的任务的数量，其默认值是 2。
+
+### tidb_build_sampling_stats_concurrency
+
+在执行分析普通列任务的时候，`tidb_build_sampling_stats_concurrency` 可以用于控制执行采样任务的并发数量，其默认值是 2。
+
+### tidb_analyze_partition_concurrency
+
+在执行 analyze 的时候，`tidb_analyze_partition_concurrency` 可以用于控制任务写入的并发度，其默认值是 2。
```
Comment (hfxsd) [inline statistics.md:394]:
```suggestion
#### tidb_build_sampling_stats_concurrency

在执行 ANALYZE 普通列任务的时候，[`tidb_build_sampling_stats_concurrency`](/system-variables.md#tidb_build_sampling_stats_concurrency-从-v750-版本开始引入) 可以用于控制执行采样任务的并发数量，其默认值是 2。

#### tidb_analyze_partition_concurrency

在执行 ANALYZE 的时候，[`tidb_analyze_partition_concurrency`](/system-variables.md#tidb_analyze_partition_concurrency) 可以用于控制任务写入的并发度，其默认值是 2。
```

File: statistics.md [hunk 2]
```diff
@@ -393,6 +401,14 @@ ANALYZE INCREMENTAL TABLE TableName PARTITION PartitionNameList INDEX [IndexName
...
 在执行分析索引列任务的时候，`tidb_index_serial_scan_concurrency` 可以用于控制一次读取的 Region 数量，其默认值是 1。
 
+
+> **注意：**
+>
+>  `tidb_build_stats_concurrency`, `tidb_build_sampling_stats_concurrency` 和 `tidb_analyze_partition_concurrency` 其实是上下游关系。
+>   以在变更这些参数的时候，需要同时考虑这三个参数的值。建议按 `tidb_analyze_partition_concurrency`，
+>   `tidb_build_sampling_stats_concurrency`， `tidb_build_stats_concurrency` 逐个调节，并观察对系统的影响。这三个参数的值越大，对系统的资源开销就越大。
+
+
```
Comment (hfxsd) [inline statistics.md:411]:
```suggestion
> **注意：**
>
>  `tidb_build_stats_concurrency`、`tidb_build_sampling_stats_concurrency` 和 `tidb_analyze_partition_concurrency` 其实是上下游关系。所以在变更这些参数的时候，需要同时考虑这三个参数的值。建议按 `tidb_analyze_partition_concurrency`、`tidb_build_sampling_stats_concurrency`、`tidb_build_stats_concurrency` 的顺序逐个调节，并观察对系统的影响。这三个参数的值越大，对系统的资源开销就越大。

```

File: statistics.md [hunk 3]
```diff
@@ -383,7 +383,15 @@ ANALYZE INCREMENTAL TABLE TableName PARTITION PartitionNameList INDEX [IndexName
 
 #### tidb_build_stats_concurrency
 
-目前 ANALYZE 执行的时候会被切分成一个个小的任务，每个任务只负责某一个列或者索引。`tidb_build_stats_concurrency` 可以控制同时执行的任务的数量，其默认值是 4。
+目前 ANALYZE 执行的时候会被切分成一个个小的任务，每个任务只负责某一个列或者索引。`tidb_build_stats_concurrency` 可以控制同时执行的任务的数量，其默认值是 2。
```
Comment (hfxsd) [inline statistics.md:386]:
```suggestion
目前 ANALYZE 执行的时候会被切分成一个个小的任务，每个任务只负责某一个列或者索引。`tidb_build_stats_concurrency` 可以控制同时执行的任务的数量，其默认值是 2。TiDB v7.4.0 及其之前版本默认值为 `4`。
```

File: statistics.md [hunk 4]
```diff
@@ -383,7 +383,15 @@ ANALYZE INCREMENTAL TABLE TableName PARTITION PartitionNameList INDEX [IndexName
...
+
+#### tidb_build_sampling_stats_concurrency
+
+在执行 ANALYZE 普通列任务的时候，[`tidb_build_sampling_stats_concurrency`](/system-variables.md#tidb_build_sampling_stats_concurrency-从-v750-版本开始引入) 可以用于控制执行采样任务的并发数量，其默认值是 2。
+
+#### tidb_analyze_partition_concurrency
+
+在执行 ANALYZE 的时候，[`tidb_analyze_partition_concurrency`](/system-variables.md#tidb_analyze_partition_concurrency) 可以用于控制任务写入的并发度，其默认值是 2。
```
Comment (hfxsd) [inline statistics.md:394]:
```suggestion
在执行 ANALYZE 的时候，[`tidb_analyze_partition_concurrency`](/system-variables.md#tidb_analyze_partition_concurrency) 可以用于控制任务写入的并发度，其默认值是 2。TiDB v7.4.0 及其之前版本默认值为 `1`。
```

File: statistics.md [hunk 5]
```diff
@@ -383,7 +377,15 @@ ANALYZE INCREMENTAL TABLE TableName PARTITION PartitionNameList INDEX [IndexName
 
 #### tidb_build_stats_concurrency
 
-目前 ANALYZE 执行的时候会被切分成一个个小的任务，每个任务只负责某一个列或者索引。`tidb_build_stats_concurrency` 可以控制同时执行的任务的数量，其默认值是 4。
+目前 ANALYZE 执行的时候会被切分成一个个小的任务，每个任务只负责某一个列或者索引。`tidb_build_stats_concurrency` 可以控制同时执行的任务的数量，其默认值是 2。TiDB v7.4.0 及其之前版本默认值为 `4`。
```
Comment (qw4990) [inline statistics.md:380]: 能补一个简单的说明图，大致介绍一下 analyze 分区表的流程、阶段吗，不然单看这几个参数，用户应该不知道他们在控制什么。。

File: statistics.md [hunk 6]
```diff
@@ -381,9 +375,24 @@ ANALYZE INCREMENTAL TABLE TableName PARTITION PartitionNameList INDEX [IndexName
...
+> ![analyze_concurrency](/media/analyze_concurrency.png)
+>
+> `tidb_build_stats_concurrency`、`tidb_build_sampling_stats_concurrency` 和 `tidb_analyze_partition_concurrency` 其实如图所示，为上下游关系, 实际的总并发为: `tidb_build_stats_concurrency`* (`tidb_build_sampling_stats_concurrency` + `tidb_analyze_partition_concurrency`) 。所以在变更这些参数的时候，需要同时考虑这三个参数的值。建议按 `tidb_analyze_partition_concurrency`、`tidb_build_sampling_stats_concurrency`、`tidb_build_stats_concurrency` 的顺序逐个调节，并观察对系统的影响。这三个参数的值越大，对系统的资源开销就越大。
+
+
 #### tidb_build_stats_concurrency
 
-目前 ANALYZE 执行的时候会被切分成一个个小的任务，每个任务只负责某一个列或者索引。`tidb_build_stats_concurrency` 可以控制同时执行的任务的数量，其默认值是 4。
+目前 ANALYZE 执行的时候会被切分成一个个小的任务，每个任务只负责某一个列或者索引。`tidb_build_stats_concurrency` 可以控制同时执行的任务的数量，其默认值是 2。TiDB v7.4.0 及其之前版本默认值为 `4`。
+
```
Comment (hfxsd) [inline statistics.md:388]:
```suggestion

![Process of ANALYZE](/media/analyze_concurrency.png)

```

File: statistics.md [hunk 7]
```diff
@@ -381,9 +375,24 @@ ANALYZE INCREMENTAL TABLE TableName PARTITION PartitionNameList INDEX [IndexName
 
 执行 ANALYZE 语句的时候，你可以通过一些参数来调整并发度，以控制对系统的影响。
 
+> **注意：**
+>  
+> ![analyze_concurrency](/media/analyze_concurrency.png)
+>
+> `tidb_build_stats_concurrency`、`tidb_build_sampling_stats_concurrency` 和 `tidb_analyze_partition_concurrency` 其实如图所示，为上下游关系, 实际的总并发为: `tidb_build_stats_concurrency`* (`tidb_build_sampling_stats_concurrency` + `tidb_analyze_partition_concurrency`) 。所以在变更这些参数的时候，需要同时考虑这三个参数的值。建议按 `tidb_analyze_partition_concurrency`、`tidb_build_sampling_stats_concurrency`、`tidb_build_stats_concurrency` 的顺序逐个调节，并观察对系统的影响。这三个参数的值越大，对系统的资源开销就越大。
+
```
Comment (hfxsd) [inline statistics.md:383]:
```suggestion
相关系统变量的参数如下图所示：

![analyze_concurrency](/media/analyze_concurrency.png)

`tidb_build_stats_concurrency`、`tidb_build_sampling_stats_concurrency` 和 `tidb_analyze_partition_concurrency` 为上下游关系。实际的总并发为：`tidb_build_stats_concurrency`* (`tidb_build_sampling_stats_concurrency` + `tidb_analyze_partition_concurrency`) 。所以在变更这些参数的时候，需要同时考虑这三个参数的值。建议按 `tidb_analyze_partition_concurrency`、`tidb_build_sampling_stats_concurrency`、`tidb_build_stats_concurrency` 的顺序逐个调节，并观察对系统的影响。这三个参数的值越大，对系统的资源开销就越大。
```

File: statistics.md (hunk 5 above)
Comment (hfxsd) [reply statistics.md:386]: 已添加 https://github.com/pingcap/docs-cn/pull/15389/files#diff-d6519e173bda79fe8ce554dc3c7638a3abe9878313e791a1358b2dc39f9fea79

File: statistics.md [hunk 8]
```diff
@@ -381,9 +375,23 @@ ANALYZE INCREMENTAL TABLE TableName PARTITION PartitionNameList INDEX [IndexName
...
+相关系统变量的关系如下图所示：
+
+![analyze_concurrency](/media/analyze_concurrency.png)
+
+`tidb_build_stats_concurrency`、`tidb_build_sampling_stats_concurrency` 和 `tidb_analyze_partition_concurrency` 为上下游关系。实际的总并发为：`tidb_build_stats_concurrency`* (`tidb_build_sampling_stats_concurrency` + `tidb_analyze_partition_concurrency`) 。所以在变更这些参数的时候，需要同时考虑这三个参数的值。建议按 `tidb_analyze_partition_concurrency`、`tidb_build_sampling_stats_concurrency`、`tidb_build_stats_concurrency` 的顺序逐个调节，并观察对系统的影响。这三个参数的值越大，对系统的资源开销就越大。
+
 #### tidb_build_stats_concurrency
 
-目前 ANALYZE 执行的时候会被切分成一个个小的任务，每个任务只负责某一个列或者索引。`tidb_build_stats_concurrency` 可以控制同时执行的任务的数量，其默认值是 4。
+目前 ANALYZE 执行的时候会被切分成一个个小的任务，每个任务只负责某一个列或者索引。`tidb_build_stats_concurrency` 可以控制同时执行的任务的数量，其默认值是 2。TiDB v7.4.0 及其之前版本默认值为 `4`。
```
Comment (hfxsd) [inline statistics.md:386]:
```suggestion
目前 `ANALYZE` 执行的时候会被切分成一个个小的任务，每个任务只负责某一个列或者索引。`tidb_build_stats_concurrency` 可以控制同时执行的任务的数量，其默认值是 `2`。TiDB v7.4.0 及其之前版本默认值为 `4`。
```

File: statistics.md [hunk 9]
```diff
@@ -381,9 +375,23 @@ ANALYZE INCREMENTAL TABLE TableName PARTITION PartitionNameList INDEX [IndexName
...
+
+#### tidb_build_sampling_stats_concurrency
+
+在执行 ANALYZE 普通列任务的时候，[`tidb_build_sampling_stats_concurrency`](/system-variables.md#tidb_build_sampling_stats_concurrency-从-v750-版本开始引入) 可以用于控制执行采样任务的并发数量，其默认值是 2。
+
+#### tidb_analyze_partition_concurrency
+
+在执行 ANALYZE 的时候，[`tidb_analyze_partition_concurrency`](/system-variables.md#tidb_analyze_partition_concurrency) 可以用于控制任务写入的并发度，其默认值是 2。TiDB v7.4.0 及其之前版本默认值为 `1`。
```
Comment (hfxsd) [inline statistics.md:394]:
```suggestion
在执行 `ANALYZE` 普通列任务的时候，[`tidb_build_sampling_stats_concurrency`](/system-variables.md#tidb_build_sampling_stats_concurrency-从-v750-版本开始引入) 可以用于控制执行采样任务的并发数量，其默认值是 `2`。

#### tidb_analyze_partition_concurrency

在执行 `ANALYZE` 的时候，[`tidb_analyze_partition_concurrency`](/system-variables.md#tidb_analyze_partition_concurrency) 可以用于控制任务写入的并发度，其默认值是 `2`。TiDB v7.4.0 及其之前版本默认值为 `1`。
```

**Review by hfxsd — APPROVED** [review]

File: statistics.md [hunk 10]
```diff
@@ -381,9 +375,23 @@ ANALYZE INCREMENTAL TABLE TableName PARTITION PartitionNameList INDEX [IndexName
 
 执行 ANALYZE 语句的时候，你可以通过一些参数来调整并发度，以控制对系统的影响。
 
+相关系统变量的关系如下图所示：
+
+![analyze_concurrency](/media/analyze_concurrency.png)
+
```
Comment (qiancai) [inline statistics.md:381]: tidb_distsql_scan_concurrency 和 tidb_index_serial_scan_concurrency 也在这个章节里，但是不在上面的图里，也不在这个公式里，是否这里单独说下

File: statistics.md [hunk 11]
```diff
@@ -381,9 +375,23 @@ ANALYZE INCREMENTAL TABLE TableName PARTITION PartitionNameList INDEX [IndexName
...
+`tidb_build_stats_concurrency`、`tidb_build_sampling_stats_concurrency` 和 `tidb_analyze_partition_concurrency` 为上下游关系。实际的总并发为：`tidb_build_stats_concurrency`* (`tidb_build_sampling_stats_concurrency` + `tidb_analyze_partition_concurrency`) 。所以在变更这些参数的时候，需要同时考虑这三个参数的值。建议按 `tidb_analyze_partition_concurrency`、`tidb_build_sampling_stats_concurrency`、`tidb_build_stats_concurrency` 的顺序逐个调节，并观察对系统的影响。这三个参数的值越大，对系统的资源开销就越大。
+
 #### tidb_build_stats_concurrency
 
-目前 ANALYZE 执行的时候会被切分成一个个小的任务，每个任务只负责某一个列或者索引。`tidb_build_stats_concurrency` 可以控制同时执行的任务的数量，其默认值是 4。
+目前 `ANALYZE` 执行的时候会被切分成一个个小的任务，每个任务只负责某一个列或者索引。`tidb_build_stats_concurrency` 可以控制同时执行的任务的数量，其默认值是 `2`。TiDB v7.4.0 及其之前版本默认值为 `4`。
```
Comment (qiancai) [inline statistics.md:386]:
```suggestion
`ANALYZE` 在执行时会被切分成一个个小任务，每个任务只负责某一个列或者索引的统计信息收集。`tidb_build_stats_concurrency` 用于控制可以同时执行的小任务的数量，其默认值是 `2`。TiDB v7.4.0 及其之前版本默认值为 `4`。
```

File: statistics.md (hunk 10 above)
Comment (hawkingrei) [reply statistics.md:381]: 这两个后面有介绍，其二，这两个对 analyze 的影响偏小

Comment (qiancai) [reply statistics.md:381]: "这两个对 analyze 的影响偏小"，这个可以在哪里说下吗，不然不太清楚这两个变量和前面 3 个变量的影响大小区别

Comment (qiancai) [reply statistics.md:381]: 比如，是否可以修改成

```suggestion
执行 ANALYZE 语句的时候，你可以通过一些参数来调整并发度，以控制对系统的影响。

- `tidb_build_stats_concurrency`

    `ANALYZE` 在执行时会被切分成一个个小任务，每个任务只负责某一个列或者索引的统计信息收集。`tidb_build_stats_concurrency` 用于控制可以同时执行的小任务的数量，其默认值是 `2`。TiDB v7.4.0 及其之前版本默认值为 `4`。

- `tidb_build_sampling_stats_concurrency`

    在执行 `ANALYZE` 普通列任务的时候，[`tidb_build_sampling_stats_concurrency`](/system-variables.md#tidb_build_sampling_stats_concurrency-从-v750-版本开始引入) 可以用于控制执行采样任务的并发数量，其默认值是 `2`。

- `tidb_analyze_partition_concurrency`

    在执行 `ANALYZE` 的时候，[`tidb_analyze_partition_concurrency`](/system-variables.md#tidb_analyze_partition_concurrency) 可以用于控制任务写入的并发度，其默认值是 `2`。TiDB v7.4.0 及其之前版本默认值为 `1`。

这 3 个关系如下图所示：

![analyze_concurrency](/media/analyze_concurrency.png)

`tidb_build_stats_concurrency`、`tidb_build_sampling_stats_concurrency` 和 `tidb_analyze_partition_concurrency` 为上下游关系。实际的总并发为：`tidb_build_stats_concurrency`* (`tidb_build_sampling_stats_concurrency` + `tidb_analyze_partition_concurrency`) 。所以在变更这些参数的时候，需要同时考虑这三个参数的值。建议按 `tidb_analyze_partition_concurrency`、`tidb_build_sampling_stats_concurrency`、`tidb_build_stats_concurrency` 的顺序逐个调节，并观察对系统的影响。这三个参数的值越大，对系统的资源开销就越大。

此外，你还可以通过以下两个变量调整并发度，但它们的对并发度的影响较小：

- `tidb_distsql_scan_concurrency`

    在执行分析普通列任务的时候，`tidb_distsql_scan_concurrency` 可以用于控制一次读取的 Region 数量，其默认值是 15。

- `tidb_index_serial_scan_concurrency`

    在执行分析索引列任务的时候，`tidb_index_serial_scan_concurrency` 可以用于控制一次读取的 Region 数量，其默认值是 1。
```

File: statistics.md [hunk 12]
```diff
@@ -381,17 +375,31 @@ ANALYZE INCREMENTAL TABLE TableName PARTITION PartitionNameList INDEX [IndexName
...
+在执行 `ANALYZE` 普通列任务的时候，[`tidb_build_sampling_stats_concurrency`](/system-variables.md#tidb_build_sampling_stats_concurrency-从-v750-版本开始引入) 可以用于控制执行采样任务的并发数量，其默认值是 `2`。
+
+#### tidb_analyze_partition_concurrency
+
+在执行 `ANALYZE` 的时候，[`tidb_analyze_partition_concurrency`](/system-variables.md#tidb_analyze_partition_concurrency) 可以用于控制任务写入的并发度，其默认值是 `2`。TiDB v7.4.0 及其之前版本默认值为 `1`。
 
 #### tidb_distsql_scan_concurrency
 
-在执行分析普通列任务的时候，`tidb_distsql_scan_concurrency` 可以用于控制一次读取的 Region 数量，其默认值是 15。
+在执行分析普通列任务的时候，[`tidb_distsql_scan_concurrency`](/system-variables.md#tidb_distsql_scan_concurrency) 可以用于控制一次读取的 Region 数量，其默认值是 15。同时其还会影响查询性能，请谨慎调整。
```
Comment (qiancai) [inline statistics.md:398]:
```suggestion
在执行分析普通列任务的时候，[`tidb_distsql_scan_concurrency`](/system-variables.md#tidb_distsql_scan_concurrency) 可以用于控制一次读取的 Region 数量，其默认值是 15。调大该变量值会影响查询性能，请谨慎调整。
```

File: statistics.md [hunk 13]
```diff
@@ -381,17 +375,31 @@ ANALYZE INCREMENTAL TABLE TableName PARTITION PartitionNameList INDEX [IndexName
...
 
 #### tidb_index_serial_scan_concurrency
 
-在执行分析索引列任务的时候，`tidb_index_serial_scan_concurrency` 可以用于控制一次读取的 Region 数量，其默认值是 1。
+在执行分析索引列任务的时候，[`tidb_index_serial_scan_concurrency`](/system-variables.md#tidb_index_serial_scan_concurrency) 可以用于控制一次读取的 Region 数量，其默认值是 1。同时其会影响查询性能，请谨慎调整。
```
Comment (qiancai) [inline statistics.md:402]:
```suggestion
在执行分析索引列任务的时候，[`tidb_index_serial_scan_concurrency`](/system-variables.md#tidb_index_serial_scan_concurrency) 可以用于控制一次读取的 Region 数量，其默认值是 1。调大该变量值会影响查询性能，请谨慎调整。
```

File: statistics.md [hunk 14]
```diff
@@ -347,17 +347,31 @@ ANALYZE TABLE TableName INDEX [IndexNameList] [WITH NUM BUCKETS|TOPN|CMSKETCH DE
 
 执行 ANALYZE 语句的时候，你可以通过一些参数来调整并发度，以控制对系统的影响。
```
Comment (qiancai) [inline statistics.md:348]:
```suggestion
执行 ANALYZE 语句的时候，你可以通过一些系统变量来调整并发度，以控制对系统的影响。
```

File: statistics.md [hunk 15]
```diff
@@ -347,17 +347,31 @@ ANALYZE TABLE TableName INDEX [IndexNameList] [WITH NUM BUCKETS|TOPN|CMSKETCH DE
...
+![analyze_concurrency](/media/analyze_concurrency.png)
 
-#### tidb_distsql_scan_concurrency
+`tidb_build_stats_concurrency`、`tidb_build_sampling_stats_concurrency` 和 `tidb_analyze_partition_concurrency` 为上下游关系。实际的总并发为：`tidb_build_stats_concurrency`* (`tidb_build_sampling_stats_concurrency` + `tidb_analyze_partition_concurrency`) 。所以在变更这些参数的时候，需要同时考虑这三个参数的值。建议按 `tidb_analyze_partition_concurrency`、`tidb_build_sampling_stats_concurrency`、`tidb_build_stats_concurrency` 的顺序逐个调节，并观察对系统的影响。这三个参数的值越大，对系统的资源开销就越大。
 
-在执行分析普通列任务的时候，`tidb_distsql_scan_concurrency` 可以用于控制一次读取的 Region 数量，其默认值是 15。
+#### `tidb_build_stats_concurrency`
 
-#### tidb_index_serial_scan_concurrency
+`ANALYZE` 在执行时会被切分成一个个小任务，每个任务只负责某一个列或者索引的统计信息收集。`tidb_build_stats_concurrency` 用于控制可以同时执行的小任务的数量，其默认值是 `2`。TiDB v7.4.0 及其之前版本默认值为 `4`。
```
Comment (qiancai) [inline statistics.md:358]:
```suggestion
`ANALYZE` 在执行时会被切分成一个个小任务，每个任务只负责某一个列或者索引的统计信息收集。[`tidb_build_stats_concurrency`](/system-variables.md#tidb_build_stats_concurrency) 用于控制可以同时执行的小任务的数量，其默认值是 `2`。TiDB v7.4.0 及其之前版本默认值为 `4`。
```

File: statistics.md [hunk 16]
```diff
@@ -347,17 +347,31 @@ ANALYZE TABLE TableName INDEX [IndexNameList] [WITH NUM BUCKETS|TOPN|CMSKETCH DE
...
 
-在执行分析索引列任务的时候，`tidb_index_serial_scan_concurrency` 可以用于控制一次读取的 Region 数量，其默认值是 1。
+#### `tidb_build_sampling_stats_concurrency`
+
+在执行 `ANALYZE` 普通列任务的时候，[`tidb_build_sampling_stats_concurrency`](/system-variables.md#tidb_build_sampling_stats_concurrency-从-v750-版本开始引入) 可以用于控制执行采样任务的并发数量，其默认值是 `2`。
+
+#### `tidb_analyze_partition_concurrency`
+
+在执行 `ANALYZE` 的时候，[`tidb_analyze_partition_concurrency`](/system-variables.md#tidb_analyze_partition_concurrency) 可以用于控制任务写入的并发度，其默认值是 `2`。TiDB v7.4.0 及其之前版本默认值为 `1`。
```
Comment (qiancai) [inline statistics.md:366]:
```suggestion
在执行 `ANALYZE` 的时候，[`tidb_analyze_partition_concurrency`](/system-variables.md#tidb_analyze_partition_concurrency) 可以用于控制对分区表统计信息进行读写的并发度，其默认值是 `2`。TiDB v7.4.0 及其之前版本默认值为 `1`。
```

**Pull request merged**